File: app/src/main/java/com/michel/deutschmacht/booklet/BookletActivity.java
package com.michel.deutschmacht.booklet;

import android.content.Context;
import android.content.Intent;

/** Helper to open the booklet at a specific lesson. */
public final class BookletActivity {
    private BookletActivity() {}

    public static Intent intent(Context c, int lesson) {
        Intent i = new Intent(c, BookletActivityImpl.class);
        i.putExtra("lesson", lesson);
        return i;
    }
}

File: app/src/main/java/com/michel/deutschmacht/booklet/BookletActivity.kt
package com.michel.deutschmacht.booklet

import android.content.Context
import android.content.Intent

/** Opens the booklet at a specific lesson. */
object BookletActivity {
    fun intent(context: Context, lesson: Int): Intent =
        Intent(context, BookletActivityImpl::class.java).putExtra("lesson", lesson)
}
